packages/utils/src/schema/getDefaultFormState.ts

```diff
@@ -177,6 +185,8 @@ export function computeDefaults<T = any, S extends StrictRJSFSchema = RJSFSchema
     defaults = mergeObjects(defaults!, schema.default as GenericObjectType) as T;
   } else if (DEFAULT_KEY in schema) {
     defaults = schema.default as unknown as T;
+  } else if (CONST_KEY in schema) {
+    defaults = schema.const as unknown as T;
```

Comment:
Const are different than defaults in that they should ALWAYS be set regardless of those values of the `experimental_defaultFormStateBehavior.emptyObjectFields` that restrict how defaults are set.